diff --git a/node_modules/gatsby-remark-embed-snippet/.index.js.swp b/node_modules/gatsby-remark-embed-snippet/.index.js.swp
new file mode 100644
index 0000000..77d3821
Binary files /dev/null and b/node_modules/gatsby-remark-embed-snippet/.index.js.swp differ
diff --git a/node_modules/gatsby-remark-embed-snippet/index.js b/node_modules/gatsby-remark-embed-snippet/index.js
index 33952e5..1bdf52a 100644
--- a/node_modules/gatsby-remark-embed-snippet/index.js
+++ b/node_modules/gatsby-remark-embed-snippet/index.js
@@ -35,7 +35,8 @@ module.exports = function (_ref, _temp) {
   var _ref2 = _temp === void 0 ? {} : _temp,
     directory = _ref2.directory;
   if (!directory) {
-    directory = path.dirname(markdownNode.fileAbsolutePath);
+    var absPath = markdownNode.fileAbsolutePath || markdownNode.internal.contentFilePath;
+    directory = path.dirname(absPath);
   }
   if (!fs.existsSync(directory)) {
     throw Error("Invalid directory specified \"" + directory + "\"");
